### DISCUSSION TITLE
3 installation of the same project using different ids

### DISCUSSION THREAD

**Discussion by gmcaixeta** [capacitor-community/sqlite#510] [General]
Hi,

I have a ionic mobile project that uses sqlite. In order to test the app, I usually create 3 apks - (development, stage, production):

`build.gradle` file:
```

    flavorDimensions "main"

    productFlavors {
        dev {
            dimension "main"
            applicationId "com.appdev"
            versionNameSuffix "-dev"
            resValue "string", "app_name", "app1"

        }
        homol {
            dimension "main"
            applicationId "com.apphomol"
            versionNameSuffix "-homol"
            resValue "string", "app_name", "app2"
        }

        prod {
            dimension "main"
            applicationId "com.appprodpoc"
            versionNameSuffix "-prod"
            resValue "string", "app_name", "app3"
        }
    }
}
```

Everything works fine, however I suspect one app may be sharing data from another app. Does it make sense ? Is there any risk of this happen since both app uses the same database name ?

**Comment by jepiqueau** [comment]
the databases are in the databases folder of your app using the appId in the capacitor.config.ts so if you have the same appId your apps share the same database. Now in the build.gradle file you give a different applicationId so look in the Device Explorer of Android Studio if 3 differents applications have been created. if yes the databases are not shared if no the databases are shared

**Comment by gmcaixeta** [comment]
Thanks for your reply @jepiqueau. 
I am still checking the problem, but it seems to be related to one mistake I made on environment variables.

I will close this since you clarify my question.

Best regards
